test/test_indicator.h

```diff
@@ -0,0 +1,32 @@
+/* Header for indicator unit tests
+ * Provides mock helpers, externs and prototypes used by
+ * `test/test_indicator.c`.
+ */
+
+#ifndef TEST_TEST_INDICATOR_H
```

Comment:
Why the TEST_TEST prefix?
```suggestion
#ifndef TEST_INDICATOR_H
```